packages/apex-node/src/tests/syncTests.ts

```diff
@@ -116,14 +116,18 @@ export class SyncTests {
         apexTestClassIdSet
       );
 
-      result.tests.forEach(item => {
-        const keyCodeCov = `${item.apexClass.id}-${item.methodName}`;
-        const perClassCov = perClassCovMap.get(keyCodeCov);
-        perClassCov.forEach(classCov =>
-          coveredApexClassIdSet.add(classCov.apexClassOrTriggerId)
-        );
-        item.perClassCoverage = perClassCov;
-      });
+      if (perClassCovMap.entries.length > 0) {
```

Comment:
Most of these lines were not changed.  To see the actual changed, diff with whitepace turned off.